<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyboardConfigDlgClass</class>
 <widget class="QWidget" name="KeyboardConfigDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Keyboard Mapping Config</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableData">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
     <column>
      <property name="text">
       <string>Key Stroke</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Hal Pin</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblKeyStroke">
       <property name="text">
        <string>Key Stroke</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtKey"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="grpModifiers">
     <property name="title">
      <string>Modifiers</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="lblHalPin">
       <property name="text">
        <string>Hal Pin</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtHalPin"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnApplyMapping">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Add Mapping</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeleteMapping">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Delete Mapping</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>tableData</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnOk</sender>
   <signal>clicked()</signal>
   <receiver>KeyboardConfigDlgClass</receiver>
   <slot>onOK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>733</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>KeyboardConfigDlgClass</receiver>
   <slot>onCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>818</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableData</sender>
   <signal>itemChanged(QTableWidgetItem*)</signal>
   <receiver>KeyboardConfigDlgClass</receiver>
   <slot>onItemChanged(QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableData</sender>
   <signal>currentItemChanged(QTableWidgetItem*,QTableWidgetItem*)</signal>
   <receiver>KeyboardConfigDlgClass</receiver>
   <slot>onCurrentItemChanged(QTableWidgetItem*,QTableWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>660</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>523</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onNewPage()</slot>
  <slot>onDeletePage()</slot>
  <slot>onNewData()</slot>
  <slot>onDeleteData()</slot>
  <slot>onOK()</slot>
  <slot>onCancel()</slot>
  <slot>onItemChanged(QTableWidgetItem*)</slot>
  <slot>onCurrentItemChanged(QTableWidgetItem*,QTableWidgetItem*)</slot>
  <slot>onPageItemChanged(QTableWidgetItem*)</slot>
  <slot>onPageCurrentItemChanged(QTableWidgetItem*,QTableWidgetItem*)</slot>
  <slot>onPageUp()</slot>
  <slot>onPageDown()</slot>
  <slot>onEditUserFonts()</slot>
 </slots>
</ui>
